Packs/CortexXDR/TestPlaybooks/Test_Playbook_-_Cortex_XDR_-_Retrieve_File_by_sha256.yml

```diff
@@ -0,0 +1,2411 @@
+          3- The 'ExtractedFiles' context key does not contain both ZIP and Unzipped file types.
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -1360,
+          "y": -235
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "247":
+    id: "247"
+    taskid: 9f04c615-3dcc-46ff-8cf8-8af3ae9562bf
+    type: condition
+    task:
+      id: 9f04c615-3dcc-46ff-8cf8-8af3ae9562bf
+      version: -1
+      name: Verify Unzipped
+      description: Verify that the 'File.Unzipped' context key has been extracted correctly.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "248"
+      Verified:
+      - "230"
+    separatecontext: false
+    conditions:
+    - label: Verified
+      condition:
+      - - operator: isTrue
+          left:
+            value:
+              complex:
+                root: File
+                accessor: Unzipped
+            iscontext: true
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -960,
+          "y": -470
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "248":
+    id: "248"
+    taskid: 55aa8d27-e4ab-458e-8e78-8ef9760766dc
+    type: regular
+    task:
+      id: 55aa8d27-e4ab-458e-8e78-8ef9760766dc
+      version: -1
+      name: Verify Context Error - Unzipped
+      description: Prints an error entry with a given message
+      scriptName: PrintErrorEntry
+      type: regular
+      iscommand: false
+      brand: ""
+    scriptarguments:
+      message:
+        simple: |-
+          The 'File.Unzipped’ context key was not extracted properly. This may indicate that one or more of the following changes have been made to the 'Cortex XDR - Retrieve File by sha256' playbook:
+          1- The file retrieval operation failed (the 'ExtractedFiles' context key is empty).
+          2- The 'lastZipFileInWarroom' input configuration was changed for the 'UnzipFile' automation used in the 'UnZip File' task.
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -960,
+          "y": -235
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "249":
+    id: "249"
+    taskid: 6fa7f817-ed88-4b18-8466-62ee68cc8910
+    type: title
+    task:
+      id: 6fa7f817-ed88-4b18-8466-62ee68cc8910
+      version: -1
+      name: Test 'Print Error'
```

Comment:
Does this branch work fine? The print error should stop the sub-playbook eventually and breaks the flow.
And even though, I assume we can skip this branch as we already ran this playbook in the new incident that you created ( and if that fails we will get an error on execution).

Reply:
Its works fine since the 'Print Error - No path was found' task error handling configuration is set to 'continue'.
![Uploading Screenshot 2023-05-04 at 13.27.50.png…]()

I think we should keep it since its the only way to really check if changes were made to the 'Print Error - No path was found' task.